### ISSUE TITLE
Hyphens not working for paths

### ISSUE THREAD

**Issue by ScalaWilliam** [atomicbits/scraml#43]
As source code that is generated includes paths directly by the look of it.

E.g. compilation gives the following error:
```
[error] ...\scraml-test-scala\target\scala-2.11\src_managed\main\io\atomicbits\scraml\rest\user\UserResource.scala:27: '=' expected but identifier found.
[error]            def up-load = new io.atomicbits.scraml.rest.user.upload.UpLoadResource(requestBuilder.withAddedPathSegment("up-load"))
[error] ...\scraml-test-scala\target\scala-2.11\src_managed\main\io\atomicbits\scraml\rest\user\UserResource.scala:29: illegal start of simple expression

```

reproduce:

```
https://github.com/atomicbits/scraml-test-scala/blob/develop/src/main/resources/io/atomicbits/scraml/user.raml#L31
```
change ```upload``` to ```up-load```

then do sbt compile.

**Comment by rigolepe** [comment]
I pushed a fix on de develop branch 0.4.4-SNAPSHOT. I'll double-check the Java version this evening and release 0.4.4 as soon as all is tested.

**Comment by rigolepe** [comment]
Fixed in 0.4.4.
